webpages/settings/index.html

```diff
@@ -61,7 +61,7 @@
           </li>
           <template v-for="preset in addon.presets">
             <li
-              v-if="preset.values.hasOwnProperty(setting.id) && preset.values[setting.id] != setting.default"
+              v-if="preset.values.hasOwnProperty(setting.id) && preset.values[setting.id].toLowerCase() != setting.default.toLowerCase()"
```

Comment:
Why?

Reply:
Some addons use uppercase and lowercase hex inconsistently, including this one.